contracts/axelar-operators/src/contract.rs

```diff
@@ -26,30 +22,17 @@ impl AxelarOperators {
         Ok(())
     }
 
-    pub fn owner(env: &Env) -> Result<Address, ContractError> {
-        env.storage()
-            .instance()
-            .get(&DataKey::Owner)
-            .ok_or(ContractError::NotInitialized)
+    pub fn owner(env: &Env) -> Address {
+        assert_some!(env.storage().instance().get(&DataKey::Owner))
```

Comment:
```suggestion
        env.storage().instance().get(&DataKey::Owner).expect("owner not found")
```
`assert`'s are meant for tests, use `expect` to make it clear that we can unwrap, and it's fine to panic otherwise